ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -241,4 +247,15 @@ public FrontendConfig(VerifiableProperties verifiableProperties) {
     maxJsonRequestSizeBytes =
         verifiableProperties.getLongInRange(MAX_JSON_REQUEST_SIZE_BYTES_KEY, 20 * 1024 * 1024, 0, Long.MAX_VALUE);
   }
+
+  /**
+   * If the string starts or ends with a slash, remove them.
+   * @param string the string to strip.
+   * @return the string with the leading and trailing slash remove.
+   */
+  private String stripLeadingAndTrailingSlash(String string) {
```

Comment:
Is this not exactly `string.replaceAll("^/|/$", "");`?

Reply:
Yep, that's cleaner. Replaced it with the regex